src/main/kotlin/no/nav/familie/ef/sak/service/AvstemmingService.kt

```diff
@@ -1,13 +1,19 @@
 package no.nav.familie.ef.sak.service
 
+import no.nav.familie.ef.sak.integration.FAGSYSTEM
+import no.nav.familie.ef.sak.integration.OppdragClient
+import no.nav.familie.kontrakter.felles.oppdrag.GrensesnittavstemmingRequest
 import org.springframework.stereotype.Service
 import java.time.LocalDateTime
 
 @Service
-class AvstemmingService {
+class AvstemmingService(val oppdragClient: OppdragClient) {
 
     fun grensesnittavstemOppdrag(fraTidspunkt: LocalDateTime, tilTidspunkt: LocalDateTime) {
-        //TODO logikk for avstemming
+        val grensesnittavstemmingRequest = GrensesnittavstemmingRequest(FAGSYSTEM, fraTidspunkt, tilTidspunkt)
```

Comment:
Er ganske sikker på at fagsystemet for avstemming er "EF" og ikke "EFOG". Du kan dobbeltsjekke det med @nav-jorn-ola-birkeland.

Reply:
Det er @nav-jorn-ola-birkeland som har lagt inn koden i forbindelse med å sende utbetalingsoppdrag, så jeg antar at den stemmer. Finer vi uansett ut av når vi begynner å teste.

Reply:
Ja, jeg tror det er forskjell på fagområdet vi setter ved avstemming og fagområdet som vi setter når vi iverksetter. Det er en ganske lang feedback-loop å få testet dette, så jeg tenker det er greit å finne ut av på forhånd - enten ved å spørre Jobi eller Steinar Hansen på økonomi

Reply:
Det slår meg at vi kanskje skal løse dette i familie-oppdrag og sende EFOG fra ef-sak da familie-oppdrag plukker oppdrag basert på fagområdet vi iverksatte på. Så må vi bare mappe EFOG -> EF når vi lager avstemmingsmeldingene.

Reply:
Høres ut som en plan